### PR TITLE
Hdrp/volume debug

### PR DESCRIPTION
**Pull request by adrien-de-tocqueville**
**DONT FORGET TO ADD A CHANGELOG**

### Checklist for PR maker
- [ ] Have you added a backport label (if needed)? For example, the `need-backport-2019.3` label. After you backport the PR, the label changes to `backported-2019.3`.
- [x] Have you updated the changelog? Each package has a `CHANGELOG.md` file.
- [x] Have you updated or added the documentation for your PR? When you add a new feature, change a property name, or change the behavior of a feature, it's best practice to include related documentation changes in the same PR.
- [ ] Have you added a graphic test for your PR (if needed)? When you add a new feature, or discover a bug that tests don't cover, please add a graphic test.

---
### Purpose of this PR
This PR is adding a new debug panel for volumes as well as a new table widget for the debug UI.

debug panel

---
### Testing status

**Manual Tests**: What did you do?
- [x] Opened test project + Run graphic tests locally
- [x] Built a player
- [ ] Checked new UI names with UX convention
- [ ] Tested UI multi-edition + Undo/Redo + Prefab overrides + Alignment in Preset
- [ ] C# and shader warnings (supress shader cache to see them)
- [ ] Checked new resources path for the reloader (in devloper mode, you have a button at end of resources that check the pathes)
- Other: 

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline/tree/HDRP%252Fvolume-debug

Any test projects to go with this to help reviewers?

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

File: com.unity.render-pipelines.core/Runtime/Volume/Volume.cs [hunk 1]
```diff
@@ -82,7 +82,7 @@ public VolumeProfile profile
             set => m_InternalProfile = value;
         }
 
-        internal VolumeProfile profileRef => m_InternalProfile == null ? sharedProfile : m_InternalProfile;
+        public VolumeProfile profileRef => m_InternalProfile == null ? sharedProfile : m_InternalProfile;
```
Comment (JulienIgnace-Unity) [inline com.unity.render-pipelines.core/Runtime/Volume/Volume.cs]: I'd like to have @Chman opinion on this. Is that dangerous?

**Comment by iM0ve** [comment]
Test status: in progress

Issues:
1) Deprecated fog components are present and should be removed.

File: com.unity.render-pipelines.core/Runtime/Volume/Volume.cs (hunk 1 above)
Comment (Chman) [reply com.unity.render-pipelines.core/Runtime/Volume/Volume.cs]: Not necessarily dangerous but honestly I'd rather avoid it. The volume API currently follows the same pattern as the Material API (with auto-instantiated profiles etc), and exposing this field will only make it more confusing, which is why we have `HasInstantiatedProfile()` in the public API. It's also missing documentation.

Comment (adrien-de-tocqueville) [reply com.unity.render-pipelines.core/Runtime/Volume/Volume.cs]: Ok, I'll use `HasInstantiatedProfile`.

**Comment by sebastienlagarde** [comment]
Poking @iM0ve for an update on QA side after all those fix commit :)

**Pull request merged**